book/running-wallaroo/running-wallaroo.md

```diff
@@ -49,35 +45,27 @@ Assuming we are running a two worker cluster on a single machine, we would run t
 {% codetabs name="Python", type="py" -%}
 *Worker 1*
```

Comment:
Looking at https://wallaroo-docs-rc.netlify.com/book/running-wallaroo/running-wallaroo.html, it seems as though there is no markdown processing at all. Given that, I'm not sure if having the `*Worker 1*` markdown formatting is something we still want or not.